### PR TITLE
COMMERCE-5280 Fix invalid sku selection render data

### PR DESCRIPTION
**Pull request by alecsloan**

### CODE REVIEW

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
@alecsloan can you please check sf?

File: modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/utilities/forms/DDMFormHandler.js [hunk 1]
```diff
@@ -47,15 +47,19 @@ class DDMFormHandler {
 			body: fieldsParam,
 			headers: new Headers({'x-csrf-token': Liferay.authToken}),
 		}).then((cpInstance) => {
-			if (cpInstance.cpInstanceExist) {
-				const dispatchedPayload = {
-					addToCartId: this.addToCartId,
-					cpInstance,
-					formFields: this.fields,
-				};
+			var dispatchedPayload = {
```
Comment (gianmarcobrunialti) [inline modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/utilities/forms/DDMFormHandler.js:50]: `const` can work here, it's an `object` whose keys are not altering the reference when re-assigned. :)

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
@ivicac Fixed SF and updated with Gian's requested changes.

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by marco-leo** [comment]
ci:test:relevant

**Comment by marco-leo** [comment]
ci:test:sf

**Comment by gianmarcobrunialti** [comment]
To be rechecked here: https://github.com/liferay-commerce/liferay-portal/pull/531

**Pull request closed without merging**